src/test/java/org/opentripplanner/model/TimetableTest.java

```diff
@@ -130,26 +109,27 @@ public void testUpdate() {
       TestUtils.dateInSeconds("America/New_York", 2009, AUGUST, 7, 0, 10, 0)
     );
     tripUpdate = tripUpdateBuilder.build();
-    tripTimesPatch = timetable.createUpdatedTripTimes(tripUpdate, timeZone, serviceDate);
-    updatedTripTimes = tripTimesPatch.getTripTimes();
-    assertNull(updatedTripTimes);
+    patch = timetable.createUpdatedTripTimes(tripUpdate, timeZone, serviceDate);
+    assertNull(patch);
 
     //---
     long startTime = TestUtils.dateInSeconds("America/New_York", 2009, AUGUST, 7, 0, 0, 0);
     long endTime;
     options.setDateTime(Instant.ofEpochSecond(startTime));
 
     //---
+    /*
```

Comment:
These lines can be removed? Or at least a todo added to move them to use RoutingWorker